### ISSUE TITLE
Include THREDDS server address for datasets

### ISSUE THREAD

**Issue by ocehugo** [aodn/content#344]
For some cases, I think It would be more straight-forward to provide the THREDDS endpoint of the file(s) at step 3.

User case:
user Joe wants to download all the SST for his region in WA. He knows how opendap works and that files are individually stored. He just wants an example opendap address for a typical file, so he can loop through the address with a regex to cut the region/time window with his favourite software (say nco).

Current behaviour: 
At the moment, the fastest way to know the address is several clicks away.
Select SST in step1, click next in step 2, click "view metadata record" in step 3, search for opendap in metadata page (which is quite dense), click in the "opendap link" in the Online Resources header, and click towards a year and get the file address.

Suggested behaviour:
Click SST in step1, click next in step2, show opendap end-point after the "Time period" information. In any case of date/product selection, I believe the opendap addresses should pop-up straight below, maybe in a hyperlink called "opendap start-point".

After the first time, some users will know that everything can be found in thredds.aodn.org.au/thredds/catalog.html. However, the portal will still made data discovery much easier.

**Comment by ggalibert** [comment]
For most collections that involve NetCDF files, the THREDDS catalog URL for this collection is already provided in step 2, info tab, in Supplementary resources. You don't need to go through the metadata record.

If it's not the case this should be fixed.

**Comment by ocehugo** [comment]
You are correct, there is a faster way to obtain it.  I just expected the address to be within the "download" step.

**Comment by ggalibert** [comment]
What we could easily do is rename "List of URLs" to "List of HTTP URLs" and similarly add a new option "List of OPeNDAP URLs".

**Comment by ocehugo** [comment]
@gg, good option, so people can avoid generating expressions or ping/ponging the server with invalid requests/filepaths.

Speaking about it, can we request a catalog/sitemap of aodn thredds files this days?

**Comment by ggalibert** [comment]
https://github.com/aodn/aodn-portal/pull/2670 is updating the label in step 3 download to make it clear that we get HTTP URLs.

**Comment by ggalibert** [comment]
Retrieving OPeNDAP URLs in download options will require changes in aodn/aodn-portal, this is not just a content issue. @pblain can we have a backlog item for that? Happy to write it for you.
